core/src/main/java/uk/gov/justice/services/core/mapping/SchemaIdMappingObserver.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.justice.services.core.mapping;
+
+import static java.lang.String.format;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import uk.gov.justice.services.core.annotation.AnyLiteral;
+import uk.gov.justice.services.core.annotation.SchemaIdMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.spi.AfterDeploymentValidation;
+import javax.enterprise.inject.spi.Bean;
+import javax.enterprise.inject.spi.BeanManager;
+import javax.enterprise.inject.spi.Extension;
+
+import org.slf4j.Logger;
+
+/**
+ * Collects all {@link MediaTypeToSchemaIdMapper} beans that are annotated with
+ * {@link SchemaIdMapper} after deployment, and allows access to list.
+ */
+public class SchemaIdMappingObserver implements Extension {
+
+    private static final Logger LOGGER = getLogger(SchemaIdMappingObserver.class);
```

Comment:
I'm guessing that you're using a static LOGGER rather than an injected one, it that this class is not managed by CDI. Otherwise the logger should be injected

Reply:
If the Logger is injected this does not work with the OpenEJB tests...maybe there is a work around...but keeping as it is for now.